core/errors.go

```diff
@@ -128,6 +128,8 @@ var (
 		txbuilder.ErrBadWitnessComponent:   errorInfo{400, "CH733", "Invalid witness component"},
 		txbuilder.ErrRejected:              errorInfo{400, "CH735", "Transaction rejected"},
 		txbuilder.ErrNoTxSighashCommitment: errorInfo{400, "CH736", "Transaction is not final, additional actions still allowed"},
+		txbuilder.ErrTxSignatureFailure:    errorInfo{400, "CH737", "Transaction signature missing, possibly missing signature key"},
+		txbuilder.ErrNoTxSighashAttempt:    errorInfo{400, "CH738", "Transaction was not attempted"},
```

Comment:
Somehow indicating that signing was what was not attempted in this error would be good.